package.json

```diff
@@ -153,11 +153,11 @@
         "preprocess": "^3.2.0",
         "prettier": "^2.5.1",
         "regenerator-runtime": "^0.13.9",
-        "sass": "^1.49.8",
+        "sass": "^1.49.9",
         "sass-loader": "^12.6.0",
         "script-loader": "0.7.2",
         "serve-static": "^1.13.2",
-        "simple-git": "^3.2.4",
```

Comment:
I'm going to ask dependabot to rebase, it looks like a simple-git bump snuck in this PR.